Gemfile

```diff
@@ -39,6 +39,8 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  # Rubocop -> Lintern
+  gem 'rubocop', require: false
```

Comment:
In general, we prefer to be explicit about the version of the gems, you can use the one that's on the gemfile.lock as a reference, in this case, you can change this line to:
```ruby
 gem 'rubocop', require: false, '~> 0.55.0'
```

Reply:
If you want to have a better understanding of versioning at gemfile, take a look at https://www.endpoint.com/blog/2013/11/07/specify-versions-for-your-dependencies

Reply:
thanks, i'll modify these files and commit